packages/web_benchmarks/lib/src/browser.dart

```diff
@@ -236,18 +237,18 @@ String _findSystemChromeExecutable() {
     return '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome';
   } else if (io.Platform.isWindows) {
     const String kWindowsExecutable = r'Google\Chrome\Application\chrome.exe';
-    final List<String> kWindowsPrefixes = <String>[
+    final List<String?> kWindowsPrefixes = <String?>[
       io.Platform.environment['LOCALAPPDATA'],
       io.Platform.environment['PROGRAMFILES'],
       io.Platform.environment['PROGRAMFILES(X86)'],
     ];
-    final String windowsPrefix = kWindowsPrefixes.firstWhere((String prefix) {
+    final String windowsPrefix = kWindowsPrefixes.firstWhere((String? prefix) {
       if (prefix == null) {
         return false;
       }
       final String expectedPath = path.join(prefix, kWindowsExecutable);
       return io.File(expectedPath).existsSync();
-    }, orElse: () => '.');
+    }, orElse: () => '.')!;
```

Comment:
This will not be necessary if you make `kWindowsPrefixes` non-nullable, and use list conditionals when populating it.